spec/bookshelf-spec.ts

```diff
@@ -633,11 +633,13 @@ describe('Bookshelf relations', () => {
     bookshelf.knex.destroy(done);
   });
 
-  it('should add relationships object', () => {
+  fit('should add relationships object', () => {
```

Comment:
that *was* the point: you were adding a test that commands jasmine to not-to-test all the other ones. Wrong in so many levels. But you ended changing it back and me not noticing. Everyone happy ;)